bin/update_pythons.py

```diff
@@ -215,9 +213,7 @@ def __init__(self) -> None:
         self.windows_64 = WindowsVersions("64")
         self.windows_pypy = PyPyVersions("32")
 
-        self.macos_6 = CPythonVersions(plat_arch="macosx_x86_64", file_ident="macosx10.6.pkg")
-        self.macos_9 = CPythonVersions(plat_arch="macosx_x86_64", file_ident="macosx10.9.pkg")
-        self.macos_u2 = CPythonVersions(plat_arch="macosx_universal2", file_ident="macos11.0.pkg")
+        self.macos_x86_64 = CPythonVersions()
```

Comment:
i guess this variable should be just `self.macos_cpython`? I don't see x86_64 parts...?

Reply:
Sure.